images/router/haproxy/reload-haproxy

```diff
@@ -80,57 +75,9 @@ done
 old_pids=$(ps -A -opid,args | grep haproxy | egrep -v -e 'grep|reload-haproxy' | awk '{print $1}' | tr '\n' ' ')
 
 reload_status=0
-installed_iptables=0
 if [ -n "$old_pids" ]; then
-  if $(set | grep DROP_SYN_DURING_RESTART= > /dev/null) && [[ "$DROP_SYN_DURING_RESTART" == 'true' || "$DROP_SYN_DURING_RESTART" == '1' ]]; then
```

Comment:
We also should remove `DROP_SYN_DURING_RESTART` traces from `hack/lib/start.sh` and `test/extended/setup.sh`.

Reply:
This also needs docs change. We need to update `architecture/topics/router_environment_variables.adoc` and `install_config/router/default_haproxy_router.adoc`.